models/staging/jira_staging/stg_rag_jira__issue.sql

```diff
@@ -57,7 +57,7 @@ final as (
         resolution as resolution_id,
         status as status_id,
         cast(status_category_changed as {{ dbt.type_timestamp() }}) as status_changed_at,
-        summary as issue_name,
+        summary as title,
```

Comment:
This needs to be documented as a breaking change in the CHANGELOG

Reply:
added